backend/src/test/java/de/neuefische/teddyscoronadiaries/controller/RecipeControllerTest.java

```diff
@@ -52,6 +52,7 @@ public void getRecipeShouldReturnRecipe(){
                 .imageUrl("some-image-url")
                 .diaryEntry("wenn absolut nichts mehr geht, dann ist es Zeit für Schokokuchen")
                 .quarantineDay(25)
+                .amountOfPeople("Für 20cm SPringform")
```

Comment:
```suggestion
                .amountOfPeople("Für 20cm Springform")
```
Hm.. if this is more general, maybe call it just `amount`?

Reply:
how about `intendedFor` ?